packages/patterns-react/src/patterns/ListSection/ListGroup.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
+import { settings } from 'carbon-components';
+import List from './List';
+
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ * List Group Component
+ *
+ * @param {object} props props object
+ * @param {string} props.title List Group title
+ * @param {string} props.lists link Group lists object
+ * @returns {object} JSX Object
+ */
+const ListGroup = ({ listGroup: { title, lists } }) => {
+  return (
+    <div
+      data-autoid={`${stablePrefix}--listgroup`}
```

Comment:
Suggest we rename to `dds--listsection-group`

Reply:
Renamed the component name to ListSectionGroup by [this commit](https://github.com/carbon-design-system/ibm-dotcom-library/pull/573/commits/b97647196df258f083f558f3cc33f81a646767e9)